x/wasm/keeper/recurse_test.go

```diff
@@ -56,9 +56,9 @@ func initRecurseContract(t *testing.T) (contract sdk.AccAddress, ctx sdk.Context
 
 func TestGasCostOnQuery(t *testing.T) {
 	const (
-		GasNoWork uint64 = 63_983
+		GasNoWork uint64 = 63_968
 		// Note: about 100 SDK gas (10k CosmWasm gas) for each round of sha256
-		GasWork50 uint64 = 64_222 // this is a little shy of 50k gas - to keep an eye on the limit
+		GasWork50 uint64 = 64_207 // this is a little shy of 50k gas - to keep an eye on the limit
```

Comment:
Why did these change?

Reply:
Figured it out: the label for the hackatom contract changed, resulting in less gas cost for loading the `ContractInfo`